authui/src/authflowv2/botprotection/recaptchav2.ts

```diff
@@ -34,6 +34,7 @@ export class RecaptchaV2Controller extends Controller {
             tokenInput.value = token;
           }
         },
+        size: "normal",
```

Comment:
For best clarity, you put your comment in the code, not in the commit message. This allows the future reader to knows your indentation without `git blame`.